bref/symfony-bridge/0.1/serverless.yaml

```diff
@@ -0,0 +1,46 @@
+# Read the documentation at https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml/
+service: symfony
+
+provider:
+    name: aws
+    # The AWS region in which to deploy (us-east-1 is the default)
+    region: us-east-1
+    # The stage of the application, e.g. dev, production, staging… ('dev' is the default)
+    stage: dev
+    runtime: provided.al2
+    environment:
+        # Symfony environment variables
+        APP_ENV: prod
+
+plugins:
+    - ./vendor/bref/bref
+
+functions:
+    # This function run the Symfony application
+    website:
+        handler: public/index.php
+        timeout: 28 # in seconds (API Gateway has a timeout of 29 seconds)
+        layers:
+            - ${bref:layer.php-80-fpm}
+        events:
+            - httpApi: '*'
+    # This function lets us run console commands in Lambda
+    console:
+        handler: bin/console
+        timeout: 120 # in seconds
+        layers:
+            - ${bref:layer.php-80} # PHP
+            - ${bref:layer.console} # The "console" layer
+
+package:
+    patterns:
```

Comment:
I would be rather have `package.exlude` and `package.include` over `package.patterns` as I find it clearer and easier to use and understand when not familiar with Serverless framework.

Reply:
`package.patterns` is the new approach, `package.exlude` (and include) is now deprecated. Because of that, I think that makes sense to go with patterns here (I should probably update the Bref documentation at some point too).

Reply:
Ok, I wasn't aware of the deprecation and just read the reasons behind that choice. 

While I find the new notation confusing, it totally make sense to provide an up-to-date and deprecation free file by default.

Reply:
> While I find the new notation confusing

I agree on that 😅